### DISCUSSION TITLE
Wrong 'enable 2FA' URL on gist.github.com

### DISCUSSION THREAD

**Discussion by xaph** [community/community#53693] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

gist.github.com has banner for enable 2FA but the button has relative path. Clicking on Enable 2FA gives 404 on gist.github.com. It works without any problem on github.com

<img width="1679" alt="Screenshot 2023-04-25 at 16 21 13" src="https://user-images.githubusercontent.com/124090/234306802-96611eb0-36b0-4e3d-8084-c428a7710045.png">

**Comment by James-Livesey** [comment]
Same for me too. It seems to redirect to https://gist.github.com/settings/account_two_factor_requirement/dismiss_banner?banner_action=dismiss, which doesn't exist.
